econml/inference/_inference.py

```diff
@@ -425,9 +431,12 @@ def effect_inference(self, X, *, T0, T1):
         pred = pred[np.arange(T0.shape[0]), ..., ind]
         pred_stderr = pred_stderr[np.arange(T0.shape[0]), ..., ind] if pred_stderr is not None else None
         # d_t=1 here since we measure the effect across all Ts
+        # Omit treatment name argument to fall back to the default
         return NormalInferenceResults(d_t=1, d_y=self.d_y, pred=pred,
                                       pred_stderr=pred_stderr,
-                                      inf_type='effect', **self._input_names)
+                                      inf_type='effect',
+                                      feature_names=self._input_names["feature_names"],
```

Comment:
I don't like that this is accessing the _input_names dict directly. This hsould instead be accessing the public cate_feature_names and cate_output_names.

Reply:
Fixed!